src/System.Diagnostics.Process/src/System/Diagnostics/Process.OSX.cs

```diff
@@ -58,7 +58,7 @@ public DateTime StartTime
                 // nanoseconds elapse from boot to that the process started) to seconds.
                 EnsureState(State.HaveId);
                 Interop.libproc.rusage_info_v3 info = Interop.libproc.proc_pid_rusage(_processId);
-                double seconds = info.ri_proc_start_abstime / NanoSecondToSecondFactor;
+                double seconds = info.ri_proc_start_abstime / (double)NanoSecondToSecondFactor;
```

Comment:
Seems reasonable.  I was even thinking afterwards "though it'd require a comment to tell people not to change it back".